advocacy_docs/partner_docs/ImpervaDataSecurityFabric/02-PartnerInformation.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Partner Information'
+description: 'Details for Imperva Data Security Fabric'
+
+---
+| &nbsp;     | &nbsp; |
+| ----------- | ----------- |
+| **Partner Name**      | Imperva       |
+| **Partner Product**   |  Imperva Data Security Fabric       |
```

Comment:
I think we should leave the word "Agent" out of any mentions of Imperva DSF throughout the guide so that we can differentiate between Imperva DSF and the Imperva Agent.